app/views/notes/_notes.html.erb

```diff
@@ -15,60 +15,3 @@
   </div>
 </div>
 <%= will_paginate notes, renderer: WillPaginate::ActionView::BootstrapLinkRenderer unless @unpaginated || (unpaginated ||= false) %>
```

Comment:
Hi, so actually we need to add a link to the new stylesheet here, so that the page is still visible. See how in this screenshot the styles are now missing? 

![](https://camo.githubusercontent.com/6d6bc3a57ea8fa41a873963b2f45ca5207c9b080/68747470733a2f2f73746f726167652e676f6f676c65617069732e636f6d2f706c6f7473322d73637265656e73686f74732f373233372d746573745f7461675f706167652e706e67)

The include should be:

```ruby
<%= stylesheet_link_tag "notes", :media => "all" %>
```

Does that make sense?